orthoseg/lib/postprocess_predictions.py

```diff
@@ -178,6 +184,22 @@ def postprocess_predictions(
         curr_input_path = curr_output_path
         output_paths.append(curr_output_path)
 
+    # If postprocessing steps are defined, the output of the prediction step
+    # (input_path) is renamed to ..._orig.gpkg
+    if dissolve or reclassify_to_neighbour_query or simplify_algorithm:
+        original_file = input_path.parent / f"{input_path.stem}_orig.gpkg"
+        os.rename(src=input_path, dst=original_file)
```

Comment:
```suggestion
        input_path.rename(original_file)
```
Most `os.` functions relating to file management have an equivalent when you use `pathlib.Path` and those are recommended to be used...